app/src/main/java/com/sunlightlabs/congress/services/RollService.java

```diff
@@ -13,14 +13,16 @@
 import java.text.ParseException;
 import java.text.SimpleDateFormat;
 import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
 import java.util.HashMap;
 import java.util.Iterator;
 import java.util.List;
 import java.util.Map;
 
 public class RollService {
 
-    public static String datetimeFormat = "yyyy-MM-dd hh:mm:ss";
+    public static String datetimeFormat = "yyyy-MM-dd HH:mm:ss";
```

Comment:
As noted above, fixes a bug where we should be using an hour of 00-24. There was a bug where votes at 12pm ET were being parsed as 12am ET (and thus 9pm PT the previous day).